docs/learn/quotas.rst

```diff
@@ -34,7 +34,7 @@ Inbound Data Filters
 --------------------
 
 In some cases the data you're receiving in Sentry is hard to filter, or you simply
-don't have the ability to update the client's application to apply the filters. Due
+don't have the ability to update the SDK's application to apply the filters. Due
```

Comment:
Sort of borderline usage of "client" in between "client/server" vs "raven client" senses here - would be more clear to say something like "update the SDK's configuration to apply ..."